src/Microsoft.DotNet.Darc/DarcLib/Models/Darc/DependencyGraph.cs

```diff
@@ -928,6 +929,40 @@ private static bool BuildContributesToNode(DependencyGraphNode node, Build poten
 
         return remotesMapping;
     }
+
+    private static async Task<string> GitShow(string gitLocation, string repoFolderPath, string commit, string fileName, ILogger logger)
```

Comment:
Is this intentionally duplicated between VmrManager and this darc command?

Reply:
I'm not sure I follow. I don't see a similar method on the `VmrManager`

Reply:
Huh...maybe there was something going on with GitHub. It was showing similar diffs in VmrManagerBase on Friday.